cli.js

```diff
@@ -1,3 +1,122 @@
 #!/usr/bin/env node
 
-require('./index.js')(process.argv[2] || 'src')
+const glob = require('fast-glob')
+const fs = require('fs')
+const fse = require('enfsensure')
+const path = require('path')
+const getI18n = require('./get-i18n.js')
+const setI18n = require('./set-i18n.js')
+
+async function get({ defaultLanguage, translationsRoot, viewsRoot }) {
+  const list = await glob([path.join(viewsRoot, '**', '*.view')], {
+    bashNative: ['linux'],
+  })
+
+  let i18n = {}
+
+  list.forEach(file => {
+    const viewI18n = getI18n({
+      defaultLanguage,
+      source: fs.readFileSync(file, 'utf-8'),
+      view: path.basename(file, '.view'),
+    })
+
+    Object.keys(viewI18n).forEach(lang => {
+      if (i18n[lang]) {
+        i18n[lang] = Object.assign(i18n[lang], viewI18n[lang])
+      } else {
+        i18n[lang] = viewI18n[lang]
+      }
+    })
+  })
+
+  fse.ensureDirSync(translationsRoot)
+
+  Object.keys(i18n).forEach(lang => {
+    fs.writeFileSync(
+      path.join(translationsRoot, `${lang}.json`),
+      JSON.stringify(i18n[lang], null, ' '),
+      'utf-8'
+    )
+  })
+}
+
+async function set({ defaultLanguage, translationsRoot, viewsRoot }) {
```

Comment:
No defaulting on the `defaultLanguage`?

Reply:
we're defining it together with the other params here https://github.com/viewstools/local/pull/3/files#diff-2cce40143051e25f811b56c79d619bf5R77

Reply:
Right you are! Sorry, didn't notice that :)

Reply:
no worries :)